roles/capi_cluster/defaults/main.yml

```
@@ -582,6 +582,14 @@ capi_cluster_release_values: >-
 # The name of the file into which the kubeconfig of the cluster should be output
 capi_cluster_kubeconfig_path: "{{ ansible_env.HOME }}/kubeconfig"
 
+# Set variables for token-based kubeconfig - service accounts/namespace/secrets
+magnum_service_account_namespace: magnum-service
+magnum_service_account_name: magnum-service-account
+magnum_service_secret_name: magnum-service-secret
```

Nit: Can we make this and other variable names a little more explanatory please
```suggestion
magnum_service_account_token_secret_name: magnum-service-secret
```